llvm/lib/Target/Xtensa/Disassembler/XtensaDisassembler.cpp

```diff
@@ -113,6 +113,36 @@ static DecodeStatus DecodeMR23RegisterClass(MCInst &Inst, uint64_t RegNo,
   return MCDisassembler::Success;
 }
 
+static const MCPhysReg FPRDecoderTable[] = {
+    Xtensa::F0,  Xtensa::F1,  Xtensa::F2,  Xtensa::F3, Xtensa::F4,  Xtensa::F5,
```

Comment:
Can assume the values are consecutive. See https://reviews.llvm.org/D67423 (TableGen `LessRecordRegister`)